addons/hr_expense/models/hr_expense.py

```diff
@@ -155,9 +155,20 @@ def _compute_state(self):
     def _compute_amount(self):
         for expense in self:
             if expense.unit_amount:
-                expense.untaxed_amount = expense.unit_amount * expense.quantity
                 taxes = expense.tax_ids.compute_all(expense.unit_amount, expense.currency_id, expense.quantity, expense.product_id, expense.employee_id.user_id.partner_id)
                 expense.total_amount = taxes.get('total_included')
+                expense.untaxed_amount = taxes.get('total_excluded')
```

Comment:
This line should be removed here :innocent:

Reply:
It works indeed without that line, I'll change it, thanks! :smile:

Reply:
@kba-odoo the change is done~